adapter-jdbc/src/main/resources/jdbc.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
You don't have to add the front matter, but you can. It will be added automatically if not.

Reply:
This is now addressed (removed). 👍🏼